configure

```diff
@@ -1128,7 +1136,7 @@ if [[ $USECUDA -eq 1 ]] ; then
     echo "Error: nvcc cuda compiler not found in $CUDA_HOME/bin/" > /dev/stderr
     exit 1
   fi
-  if [ -z "$SHADER_MODEL" ] ; then
+  if [ -z "$NVCCFLAGS" -a -z "$SHADER_MODEL" ] ; then
```

Comment:
The reason I initially used SHADER_MODEL was precisely to restrict execution to a single GPU shader model version; each additional architecture increases compilation time (granted not a huge deal currently for cpptraj since there are only 2 CUDA files), and when I was initially running the cpptraj GPU code I would only need it for a specific architecture. The NVCCFLAGS variable was my way of getting around this limitation if I needed it. Since the CUDA code is essentially going live this release it makes sense to have the multiple architectures in there if SHADER_MODEL isn't defined. I'll probably add that in a separate PR.